tests/contrib/operators/test_ssh_operator.py

```diff
@@ -87,15 +86,15 @@ def test_json_command_execution(self):
 
         self.assertIsNotNone(task)
 
-        ti = TaskInstance(
-            task=task, execution_date=timezone.utcnow())
-        ti.run()
-        self.assertIsNotNone(ti.duration)
-        self.assertEqual(ti.xcom_pull(task_ids='test', key='return_value'),
-                         b64encode(b'airflow').decode('utf-8'))
+        with conf_vars({('core', 'enable_xcom_pickling'): 'False'}):
+            ti = TaskInstance(
+                task=task, execution_date=timezone.utcnow())
+            ti.run()
+            self.assertIsNotNone(ti.duration)
+            self.assertEqual(ti.xcom_pull(task_ids='test', key='return_value'),
+                             b64encode(b'airflow').decode('utf-8'))
 
     def test_pickle_command_execution(self):
-        configuration.conf.set("core", "enable_xcom_pickling", "True")
```

Comment:
Use the decorators here too?

(I think I prefer the decorator approach as it means the code is indented less.)